terraso_backend/apps/graphql/schema/schema.graphql

```diff
@@ -556,6 +556,61 @@ type StoryMapNode implements Node {
   createdBy: UserNode
   isPublished: Boolean!
   publishedAt: DateTime
+  membershipList: CollaborationMembershipListNode
+}
+
+type CollaborationMembershipListNode implements Node {
+  membershipType: CollaborationMembershipListMembershipTypeChoices!
+  memberships(offset: Int, before: String, after: String, first: Int, last: Int, user: ID, user_In: [ID], userRole: String, user_Email_Icontains: String, user_Email_In: [String], membershipStatus: CollaborationMembershipMembershipStatusChoices, user_Email_Not: String): CollaborationMembershipNodeConnection!
+  id: ID!
+  accountMembership: CollaborationMembershipNode
+  membershipsCount: Int
```

Comment:
This one is a little tricky, this is something that we have also in the current group implementation, let me try to explain the issue. `memberships { totalCount }` is the total count of memberships that graph QL can return, that means that if the user can get a totalCount of 3 they should also be able to fetch the 3 memberships, also it will be linked to the filters you use for memberships (`memberships(filters) { totalCount }`) but there is also a requirement to fetch the count of memberships without giving permission to the user to actually see what those memberships are, that is what `membershipsCount` is doing, it is giving us a way to just get the number without giving access to the actual memberships data

Reply:
ahh ok i see, that distinction makes sense. in the code, anonymous users always see a membership count of 0, so i guess that distinction would matter for logged in users depending whether they are or are not a member of a closed group?

Reply:
From what I see in the code it seems that membershipCount is not 0 for landscapes, and it also just returns the count for approved memberships, so I pontential imrpovement could be to change the name, and I think there is a bug with this count for groups, but I think that value is not being used